tests/integration/test_onprem_open.py

```diff
@@ -11,13 +11,6 @@
 logger = logging.getLogger(__name__)
 
 daacs_list: list[TestParam] = [
-    {
-        "provider_name": "NSIDC_ECS",
-        "n_for_top_collections": 3,
-        "granules_count": 100,
-        "granules_sample_size": 2,
-        "granules_max_size_mb": 100,
-    },
```

Comment:
Is there no more NSIDC ECS provider? Some other PRs (#161) have failing integration tests and I was wondering if this was the root cause.

Reply:
The more popular collections were hidden recently they are still available but only with ACLs so I thinkwe better test only cloud collections.